source/agora/test/Quorum.d

```diff
@@ -53,7 +54,7 @@ unittest
     // generate 18 blocks, 1 short of the enrollments expiring.
     network.generateBlocks(Height(GenesisValidatorCycle - 2));
 
-    const keys = nodes.map!(node => node.getPublicKey())
+    const keys = nodes.map!(node => node.getPublicKey(PublicKey.init).key)
```

Comment:
Maybe a default value on the API would be reasonable here ?

Reply:
LocalRest does not like default values. We could do it in `NetworkClient` but doesnt help the tests I guess.

Reply:
Oh we should probably have it implemented in LocalRest.

Reply:
But actually, it shouldn't need special handling if we keep a reference to the interface, does it ?

Reply:
Seems like it, see the last commit.